readyness.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-if freshclam | grep -q 'bytecode.cvd is up to date'; then
+if freshclam | grep -q 'bytecode.* is up to date'; then
```

Comment:
Shouldn't it be?:
```bash
if freshclam | grep -q 'bytecode\..* is up to date'; then
```
Or this if I understand the problem:
```bash
if freshclam | grep -Eq 'bytecode\.(cvd|cvl) is up to date'; then
```
(Or perhaps I'm just too tired at this point!)

I guess it probably doesn't matter...